vispy/gloo/glir.py

```diff
@@ -876,18 +905,22 @@ def set_interpolation(self, min, mag):
 class GlirTexture2D(GlirTexture):
     _target = gl.GL_TEXTURE_2D
     
-    def set_size(self, shape, format):
+    def set_size(self, shape, format, internalformat):
```

Comment:
I think it is unnecessary unless there is some way that this GLIR implementation would be running with an older gloo that generates the older RESIZE command tuple.  The new code will always have the internalformat slot and it may hold a None value.